designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/NuResourcesTest.scala

```diff
@@ -34,52 +38,50 @@ import pl.touk.nussknacker.ui.config.FeatureTogglesConfig
 import pl.touk.nussknacker.ui.process.ProcessService.UpdateProcessCommand
 import pl.touk.nussknacker.ui.process._
 import pl.touk.nussknacker.ui.process.deployment._
+import pl.touk.nussknacker.ui.process.fragment.DbFragmentRepository
 import pl.touk.nussknacker.ui.process.marshall.ProcessConverter
 import pl.touk.nussknacker.ui.process.processingtypedata.{DefaultProcessingTypeDeploymentService, ProcessingTypeDataProvider, ProcessingTypeDataReader}
 import pl.touk.nussknacker.ui.process.repository.ProcessRepository.CreateProcessAction
 import pl.touk.nussknacker.ui.process.repository._
-import pl.touk.nussknacker.ui.process.fragment.DbFragmentRepository
 import pl.touk.nussknacker.ui.process.test.{PreliminaryScenarioTestDataSerDe, ScenarioTestService}
 import pl.touk.nussknacker.ui.processreport.ProcessCounter
 import pl.touk.nussknacker.ui.security.api.LoggedUser
 import pl.touk.nussknacker.ui.util.{ConfigWithScalaVersion, MultipartUtils}
 import slick.dbio.DBIOAction
-import _root_.sttp.client3.SttpBackend
-import _root_.sttp.client3.akkahttp.AkkaHttpBackend
-import pl.touk.nussknacker.engine.api.deployment.simple.SimpleStateStatus
-import pl.touk.nussknacker.engine.definition.test.{ModelDataTestInfoProvider, TestInfoProvider}
 
 import java.net.URI
 import scala.concurrent.{ExecutionContext, Future}
 
-trait EspItTest extends LazyLogging with WithHsqlDbTesting with TestPermissions with BeforeAndAfterEach {
+@deprecated("Consider using NuItTest with NuScenarioConfigurationHelper instead. Will be removed in the future.")
```

Comment:
We rather don't add deprecations for internal purposes. Instead, we add `TODO`. Thanks to we have deprecation warnings silenced in only those places that are necessary and see new deprecations that we have to fix. WTDY about this approach?

Reply:
I prefer to use @deprecation because IMO it's much more useful. Eg. when the deprecated class/method is used in a newly created code, the author of the code has a chance to notice that he should not use the deprecated thing. It doesn't apply to the comment-based approach - the coder can easily miss the info that the code is deprecated and use it in a new code. 

But I'll do it in the way you propose because of the cross-compilation I cannot use @nowarn annotation (yeah, silencer could be the solution)